contracts/consensus-gateway/ERC20GatewayBase.sol

```diff
@@ -49,4 +53,31 @@ contract ERC20GatewayBase {
             )
         );
     }
+
+
+    /**
+     * @notice It returns hash of withdraw intent.
+     *
+     * @param _amount Amount of tokens.
+     * @param _beneficiary Beneficiary address.
+     *
+     * @return withdrawIntentHash_ Hash of withdraw intent.
+     */
+    function hashWithdrawIntent(
+        uint256 _amount,
+        address _beneficiary
+    )
+        public
```

Comment:
Should this be internal ?

Reply:
`hashDepositIntent` is already `public`. Hence made `hashWithdrawIntent` `public`.